Gemfile

```diff
@@ -2,3 +2,5 @@ source 'http://rubygems.org'
 gemspec
 
 gem 'irb'
+gem 'concurrent-ruby', require: 'concurrent'
+gem 'webmock'
```

Comment:
not sure the best way to specify dev dependencies

Reply:
Generally wrapped in an environment:

```
group :development do
  gem 'foobar'
```

```
group :development, :test do
  gem 'foobar'
```